CHANGES

```diff
@@ -120,6 +115,123 @@ Tools
   Verbose Compat vernacular, since these warnings can now be silenced or
   turned into errors using "-w".
 
+Bugfixes
+
```

Comment:
One possibility could be to have a separate file, say BUGFIXES or so, and in which the list could go. Then, the CHANGES could have a bug section telling to look at this separate file. What do you think?